packages/fake-data/src/fakePoolCandidates.ts

```diff
@@ -10,10 +10,27 @@ import {
   Pool,
   User,
 } from "@gc-digital-talent/graphql";
-
+import {
+  GeneratedAwardExperience,
+  GeneratedCommunityExperience,
+  GeneratedEducationExperience,
+  GeneratedPersonalExperience,
+  GeneratedWorkExperience,
+} from "./fakeExperiences";
 import fakePools from "./fakePools";
 import fakeUsers from "./fakeUsers";
 
+export type GeneratedPoolCandidate = PoolCandidate & {
+  __typename: "PoolCandidate";
+  educationRequirementExperiences: Array<
+    | GeneratedAwardExperience
+    | GeneratedCommunityExperience
+    | GeneratedEducationExperience
+    | GeneratedPersonalExperience
+    | GeneratedWorkExperience
+  >;
+};
+
```

Comment:
I think this is the source of the dependency cycle. `fakeUsers.ts` imports this then `fakePoolCandidates` imports something from `fakeUsers.ts`

Reply:
Thanks!  I missed this warning.  I just moved the type definition to its own file.
Ref: [move fake type out to own file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7447/commits/2f65b30da36d599e100ec515337e842a2488b64a)